openshift/mlflow-experiment-job-template.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-experiment-job-template
+
+parameters:
+- description: Application name
+  value: 'mlflow-experiment'
+  name: APPLICATION_NAME
+  required: true
+- name: MLFLOW_TRACKING_URI
+  value: http://mlflow-server-test-route-aiops-kubeflow.cloud.paas.upshift.redhat.com/
+  required: false
```

Comment:
We should make this `required: true` because we want the user of mlflow to specify their tracking server address.

Reply:
I will change it to required and change its default value